king_phisher/plugins.py

```diff
@@ -135,6 +136,15 @@ class OptionString(OptionBase):
 	"""A plugin option which is represented with a string value."""
 	pass
 
+Requirement = collections.namedtuple('Requirement', ('type', 'value', 'comptabile'))
+
+_RequirementMeta = collections.namedtuple('RequirementMeta', ('name', 'title'))
+class _RequirementType(enum.Enum):
+	PACKAGE = _RequirementMeta(name='package', title='Required Package')
+	PYTHON_VERSION = _RequirementMeta('minimum-python-version', 'Minimum Python Version')
+	VERSION = _RequirementMeta('minimum-kp-version', 'Minimum King Phisher Version')
```

Comment:
Please change the `type` here to `minimum-version`. I know it's a little ambiguous and in hind sight it should have been king-phisher-version but to be consistent with the source key, let's keep it at `minimum-version`.